doc/api/buffer.md

```diff
@@ -1150,6 +1150,30 @@ console.log(utf16Buffer.indexOf('\u03a3', 0, 'ucs2'));
 console.log(utf16Buffer.indexOf('\u03a3', -4, 'ucs2'));
 ```
 
+If `value` is not a string, number, or Buffer, this method will throw a
```

Comment:
@nodejs/documentation: Should "Buffer" be surrounded in back-ticks in this line?